src/surge-fx/CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ target_sources(${PROJECT_NAME} PRIVATE
 target_link_libraries(${PROJECT_NAME} PRIVATE
   surge::surge-common
   surge-fx-binary
-  surge-juce
```

Comment:
i'm not entirely sure this is ok, it works but I don't fully understand the implications of this change. 

CMake is not my strongest language.